src/lib/markbind/src/parser.js

```diff
@@ -353,6 +353,11 @@ class Parser {
 
     componentParser.postParseComponents(node, this._onError);
 
+    // If a fixed header is applied to this page, generate dummy spans as anchor points
+    if (config.fixedHeader && (/^h[1-6]$/).test(node.name) && node.attribs.id) {
```

Comment:
since `(/^h[1-6]$/).test(node.name)` is used twice in the method, perhaps we could extract that out into a variable `isHeaderTag`